apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -241,7 +241,11 @@ defmodule SiteWeb.ScheduleView do
   @spec route_header_text(Route.t()) :: [String.t()] | Safe.t()
   def route_header_text(%Route{type: 3, name: name} = route) do
     if Route.silver_line?(route) do
-      ["Silver Line ", name]
+      if route.id == "746" do
+        "Silver Line Waterfront"
+      else
+        ["Silver Line ", name]
+      end
```

Comment:
just out of general curiosity why is this route handled differently from the other silver line routes?

Reply:
Using the current handling the header would be `Silver Line SLW`. It was requested to be `Silver Line Waterfront`